eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryTest.java

```diff
@@ -339,6 +339,16 @@ public void testFindByRegionsAndComplexFiltersExcludeSingleRoot() {
         }
     }
 
+    @Test
+    public void testFindDistinctChromosomesByStudyId() {
+        List<String> chromosomeList = variantEntityRepository.findDistinctChromosomes();
+
+        List<String> expectedChromosomeList = new ArrayList<>();
+        expectedChromosomeList.add("11");
```

Comment:
If other tests are not affected, adding some variants for other chromosomes to the test data json file would be nice to have other chromosomes to get a more complex test. If adding more data affects too much other tests, I would trust in the Spring distinct query and wont spend a lot of time here